docs/odh/grafana/README.md

```diff
@@ -0,0 +1,25 @@
+## Grafana Monitoring Dashboards
+
+[Grafana](https://grafana.com/oss/grafana) is a multi-platform open source analytics and interactive visualization tool. It allows you to query, create dashboards, explore, and alert on metrics when connected to supported data sources.
+
+For each of the Operate First managed services and applications, we have created example Grafana dashboards which can be used to visualize our [SLI/SLOs](https://github.com/operate-first/SRE/tree/master/sli-slo) defined for each of the applications.
```

Comment:
I would rephrase this to say: 
"For various Operate First managed services and applications..."

The distinction being, that in the original version, we're saying we already have dashboards for _all_ the operate-first managed services, which is not the case. Hope that makes sense.

Reply:
yeah that makes sense, updated it :+1: